public/locales/en/translation.json

```diff
@@ -79,5 +79,300 @@
   },
   "prefixes": {
     "error": "Error: "
+  },
+  "traits": {
+    "all": "All",
+    "HeadAccessory": {
+      "name": "Head Accessory",
+      "values": {
+        "mask": "Mask",
+        "hairclip": "Hair Clip",
+        "hairbow": "Hair Bow",
+        "covidmask": "Covid Mask",
+        "catgirl": "Cat Girl",
```

Comment:
I feel like Catgirl should be one word throughout, thoughts @0xKiwi?

Reply:
Yeah sounds good